src/olympia/amo/templates/amo/robots.html

```diff
@@ -11,9 +11,10 @@
 {% endfor %}
 
 {# Spacing is a little weird because it makes the output prettier #}
-{% for l in settings.AMO_LANGUAGES %}
-{% for a in apps -%}
+{% for l in settings.AMO_LANGUAGES -%}
 
+Allow: /{{ l }}/firefox{{ url('collections.list', add_prefix=False) }}mozilla/
```

Comment:
collection urls aren't going to have the username (i.e. `'mozilla/'`) in them once #8302 lands.

Reply:
Ah, right. Should we hardcode the mozilla user ID here (here or in `settings`, similar to `TASK_USER_ID`)?

Reply:
afaik the mozilla user id _is_  `TASK_USER_ID` so you can use that reference.